shell/platform/tizen/tizen_log_stub.cc

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <stdio.h>
+#include "tizen_log.h"
+
+int dlog_print(log_priority prio, const char* tag, const char* fmt, ...) {
+  va_list arglist;
+  va_start(arglist, fmt);
+  printf(fmt, arglist);
+  va_end(arglist);
```

Comment:
You should use vprintf(or vsprintf, vsnprintf), and don't forget add newline like dlog does

```suggestion
  va_list arglist;
  va_start(arglist, fmt);
  vprintf(fmt, arglist);
  va_end(arglist);
  printf("\n");
```

Reply:
Oh! Thanks!